src/agent/onefuzz/src/expand.rs

```diff
@@ -518,12 +509,11 @@ mod tests {
         Ok(())
     }
 
-    #[test]
-    fn test_expand_machine_id() -> Result<()> {
-        // verify {machine_id} expands to a UUID, but don't worry about the
-        // actual value of the UUID
-        let result = Expand::new().evaluate_value("{machine_id}")?;
-        Uuid::parse_str(&result)?;
+    #[tokio::test]
+    async fn test_expand_machine_id() -> Result<()> {
+        let expand = Expand::new().machine_id().await?;
+        let expanded = expand.evaluate_value("{machine_id}")?;
+        Uuid::parse_str(&expanded)?;
```

Comment:
Let's preserve the comment, for future readers!

```suggestion
        // Check that "{machine_id}" expands to a valid UUID, but don't worry about the actual value.
        Uuid::parse_str(&expanded)?;
```